tools/kdc/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mesosphere/kdc:latest
+
+RUN mkdir /kdc
+COPY run.sh /kdc/run.sh
+COPY kdc.conf /etc/heimdal-kdc/kdc.conf
+RUN chown -R nobody:nogroup /kdc
+RUN chmod -R 777 /var/lib/heimdal-kdc/
```

Comment:
These permissions seem a little extreme, but I'm ok with them as is for the time being.